### DISCUSSION TITLE
Setting managedNamespaceMetadata from CLI

### DISCUSSION THREAD

**Discussion by msobkowiak-olx** [argoproj/argo-cd#17038] [General]
Hey !

I am not able to work this out with official documentation (it does not seem to be supported) so wanted to ask - does ArgoCD CLI allow setting default annotations/labels on managed namespace via CLI ? According to --sync-policy flag format it is not possible.

**Comment by crenshaw-dev** [comment]
I don't think it's possible right now. Created an enhancement request: https://github.com/argoproj/argo-cd/issues/17039

**Reply by msobkowiak-olx** [reply]
Wanted to say - I was able to do it with CLI, you can document it and perhaps close the issue - this works with us:

`argocd --grpc-web app patch "$ARGOCD_APP_NAME" --type merge --patch '{"spec": {"syncPolicy": {"managedNamespaceMetadata": {"labels": {"elbv2.k8s.aws/pod-readiness-gate-inject": "enabled"}}}}}'`
